config/routes.rb

```diff
@@ -198,7 +196,6 @@
     # Keeping the urls to old guidance urls, but they need to lead to new api-reference ones
     get "/guidance/home", to: redirect("/api-reference")
     get "/guidance/ecf-usage", to: redirect("/api-reference/ecf-usage")
```

Comment:
A bunch of these other pages don't seem to exist either, we should maybe ditch the redirects at some point. There are also some translations we could bin here

https://github.com/DFE-Digital/early-careers-framework/blob/main/config/locales/en.yml#L600

Reply:
I think let's keep the other redirects for now as not sure about those for now and they are not NPQ related. 
In terms of other translations I think we need to do another sweep after those tickets to see whats left and break down into tickets to avoid too many changes